docs/libcurl/symbols-in-versions

```diff
@@ -968,6 +968,7 @@ CURL_VERSION_SSL                7.10
 CURL_VERSION_SSPI               7.13.2
 CURL_VERSION_TLSAUTH_SRP        7.21.4
 CURL_VERSION_UNIX_SOCKETS       7.40.0
+CURL_VERSION_ZSTD               7.71.0
```

Comment:
The feature freeze is this Wednesday (May 27th) so I doubt we'll have this ready in time. I think we need to aim to get this ready and merged in the next feature window instead.

Reply:
I plan no more modification myself on this feature.
The C code compile on several platform, test396 and 397 perform ok,
The last problem is CMake stuff

Reply:
We are May 28th, so I suppose I can replace 7.71 added by 7.72 ?

Reply:
Ah yes, good call!